### ISSUE TITLE
[queries] Current Status/Forecasting:

### ISSUE THREAD

**Issue by krobasky** [RENCI/ticpmd#4]
Current Status/Forecasting:

- [ ]       Number <per service> ongoing now
- [ ]       Number <per service> expected to start within next 3 months (maybe allow for the months to vary)
- [ ]       Number comprehensive consultations ongoing
- [ ]       Number comprehensive consultations expected to have grant approval in next 3 months (maybe allow for months to vary)
- [ ]       Number Implementations ongoing (include pilots)

These should be broken out by TIC/RIC

**Comment by rajshruti18** [comment]
Answer for this query:

1. Go to CTMD's proposal page**-->** Select ID, Proposal Name, Resource Application Status (filtered by keyword-'ongoing') and 'Assigned TIC/RIC'**-->**Export as csv
2. Cannot be answered
3. Go to CTMD's proposal page**-->** Select ID, Proposal Name, Resource Application Status (filtered by keyword-'Comprehensive Consultation Ongoing' and 'Assigned TIC/RIC'**-->**Export as csv
4. Go to CTMD's proposal page**-->** Select ID, Proposal Name, Approved for Comprehensive Consultation, Estimated Funding Start Date  and 'Assigned TIC/RIC'**-->**Export as csv**-->**Filter dates upto next 3 months from current date
5. Go to CTMD's proposal page**-->** Select ID, Proposal Name, Resource Application Status (filtered by keyword-'Implementation Ongoing' and 'Assigned TIC/RIC'**-->**Export as csv

**Comment by rajshruti18** [comment]
@krobasky Please review the answer to this query and let me now if the answers are correct.

**Comment by krobasky** [comment]
reassigning to Matt for planning out what else needs to be done to add this query to the dashboard.

**Comment by mbwatson** [comment]
i'm happy to add a feature to answer this remaining query. how do we determine this answer from the data in our database?

**Comment by mbwatson** [comment]
@rajshruti18 this is pretty old. did we ever get this last query answered?

**Comment by rajshruti18** [comment]
> @rajshruti18 this is pretty old. did we ever get this last query answered?

No, we did not. I'll need to check if there is any REDCap field that can answer it.

**Comment by dcarsey** [comment]
Approval Release Diff (tickbox on proposals tab) addresses this?  If so, is this right?

PROPOSALS tab - click the following tickboxes:

- ID
- PROPOSAL NAME
- APPROVED FOR COMPREHENSIVE CONSULTATION
- ESTIMATED FUNDING START DATE
- ASSIGNED TIC/RIC

EXPORT AS CSV
FILTER DATES UP TO 3 MONTHS FROM CURRENT DATE

Confirm w/ Shruti.
